core/trino-main/src/main/java/io/trino/simd/BlockEncodingSimdSupport.java

```diff
@@ -139,15 +140,20 @@ enum X86SimdInstructionSet {
             }
         }
 
+        // Unclear which x86_64 platforms would lack 256 bit SIMD registers, so disable vectorization
+        // because it's likely that intrinsics are missing or perform worse
         if (preferredVectorBitWidth < MINIMUM_SIMD_LENGTH) {
```

Comment:
Can we rename MINIMUM_SIMD_LENGTH to MINIMUM_SIMD_LENGTH_x86_64 and define another MINIMUM_SIMD_LENGTH_ARM for the `128` there ?
It's a bit confusing to see the static variable here and `128` in the ARM code path